SalesforceCommerceCloud/pwa-kit
# [V2] Remove deprecations

This PR removes items that have been flagged for deprecation before V2.

The removals include:
* Legacy remote proxy which allowed MRT environments to fall back and use proxies defined in package.json
* Cache-hache-manifest script, which is dead code no longer in use

Related PRs:
- #205 
- #290 
- #300 

# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# How to Test-Drive This PR

Legacy proxy removal
- Verify that proxies still run locally by starting the app
- In MRT:
> - If proxies are not set, verify you see a 501 error page with a warning to set proxies
> - If proxies are set, verify the store is functioning normally

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
